deploy/overlays/dev/patch_esp.yaml

```diff
@@ -12,6 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+- op: add
+  path: /spec/template/spec/containers/0/args/0
```

Comment:
for my understanding, what are these indexes - 0/args/0, 1/args/0 ?

Reply:
These corresponding to the deployment.yaml json tree. For example, here refers to the first container in deployment.yaml, and add an argument in the first position: https://github.com/datacommonsorg/mixer/blob/master/deploy/base/deployment.yaml#L53

Reply:
Wow, seems brittle. Add a comment?

Reply:
Done. And cleaned up the args